sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/implementation/OpenAIClientImpl.java

```diff
@@ -811,7 +809,7 @@ public Mono<Response<BinaryData>> getChatCompletionsWithResponseAsync(
      *                 content: String (Optional)
      *             }
      *             index: int (Required)
-     *             finish_reason: String(stopped/tokenLimitReached/contentFiltered) (Required)
+     *             finish_reason: String(stop/length/content_filter) (Required)
```

Comment:
@srnagar Can we use under score? Should we customize the value through TypeSpec?